package.json

```diff
@@ -11,8 +11,7 @@
     "eslint:check": "node_modules/eslint/bin/eslint.js \"src/**/*.jsx\"",
     "check:lint-prettier": "npm run eslint:check && npm run prettier:check",
     "pretest": "npm run check:lint-prettier",
-    "test": "jest --coverage",
-    "update-snapshot": "jest --coverage -u",
+    "test": "jest --coverage -u",
```

Comment:
@McHardex Can we please return our test scripts to the way we had it previously? with this configuration each time we run `npm test`  our snapshot test cases will be updated and we wont know when an unwanted change has occurred on the dom.